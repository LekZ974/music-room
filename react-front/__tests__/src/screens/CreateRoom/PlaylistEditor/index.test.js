import React from 'react';

import { shallow } from 'enzyme'
import ConnectedPlayListEditorScreen, { PlayListEditorScreen } from '../../../../../src/screens/CreateRoom/PlayListEditor';
import toJson from 'enzyme-to-json';

jest.mock('../../../../../src/i18n', () => {});
jest.mock('react-native-gesture-handler', () => {});

describe('>>>> PlayListEditorScreen - REACT-REDUX (Shallow + passing the {store} directly', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ConnectedPlayListEditorScreen />)
  })
  it('renders without crashing', () => {
    expect(wrapper.length).toEqual(1)
  })
  it('Snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})