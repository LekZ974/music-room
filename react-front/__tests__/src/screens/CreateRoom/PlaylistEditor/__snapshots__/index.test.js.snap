// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>> PlayListEditorScreen - REACT-REDUX (Shallow + passing the {store} directly Snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
